app/views/evidence/_form.html.erb

```diff
@@ -19,7 +26,7 @@
   </div>
 
   <% if @evidence.persisted? %>
-    <%= hidden_field :evidence, :original_updated_at, value: @evidence.updated_at.to_i %>
+    <%= hidden_field :evidence, :original_updated_at, value: @evidence.updated_at.to_i, data: { behavior: 'updated-as-auto-save' } %>
```

Comment:
If this is an "auto save" related behaviour, let's lead with that on the behaviour name, perhaps "auto-save-updated-as"

Reply:
It was not introduced *as* auto save behavior. It's existing conflict resolution behavior that auto-save continues to play well with.

Reply:
I'm not going to buy this one. This `data-behavior` is handled by the AutoSave code. As Such, I'd think it conventional to start with the name of the module that handles it.

Reply:
I'll bite on that completely.

Reply:
Updated to `auto-save-updated-at`